### DISCUSSION TITLE
Vote about Interrogator

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/stable-diffusion-webui-forge#1054] [Polls]
Interrogator in webui is super old.  Do we just replace BLIP and DeepDanbooru with Florence2 and WD tagger V3?

**Comment by LeDXIII** [comment]
Yes,Yes, very big YES!!!

**Comment by bkrew** [comment]
Would it be possible to make it flexible ?
Swapping the model to a suitable version that is needed, so that everyone could use a model of their choosing and hardware configuration.

**Comment by SunGreen777** [comment]
Perhaps Joy Caption Alpha Two is better? But I would replace the menu for selecting items with icons.
https://huggingface.co/spaces/fancyfeast/joy-caption-alpha-two

Sorry, off topic, no information about ControlNet?
